addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -468,6 +468,26 @@ const actions = {
         env.call('bus_service', 'startPolling');
         dispatch('_startLoopFetchPartnerImStatus');
     },
+    /**
+     *
+     * @param unused
+     * @param {Object} ev
+     * @param {string} composerLocalId
+     */
+    insertEmoji(
+        {state},
+        ev,
+        composerLocalId,
+    ) {
+        Object.assign(state.composers[composerLocalId], {
+            textInputContent : mailUtils.insertTextContent(
+                state.composers[composerLocalId].textInputContent,
```

Comment:
Due to the repetition of `state.composers[composerLocalId]` here I would store it in a variable `composer`.

Reply:
Done